<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MotoUDP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>UDP connection</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxIpAddress">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>192.168.1.14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LocalHost</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>CONNECT</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditport">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;10040&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>351</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Robot Control</string>
    </property>
    <widget class="QPushButton" name="pushButtonServo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>SERVO ON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonGetPosition">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>GET POSITION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMOVJ">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>MOV TO POSTION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMOVL">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>MOV TO PULSE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonGetPulse">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>GET PULSE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMOVC">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>MOVC</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>-1</y>
      <width>621</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Sent Data</string>
    </property>
    <widget class="QTextBrowser" name="textBrowserSentData">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>611</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>621</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Received Data</string>
    </property>
    <widget class="QTextBrowser" name="textBrowserReceiveData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>611</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>621</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Current Position</string>
    </property>
    <widget class="QTextBrowser" name="textBrowserX">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserY">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserZ">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserRY">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RZ</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserRX">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserRZ">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>510</y>
      <width>621</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Current Pulse</string>
    </property>
    <widget class="QTextBrowser" name="textBrowserS">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R2</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserL">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R3</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserR">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserB">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R6</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserU">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserT">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxGoalPos">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>191</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Goal Position</string>
    </property>
    <widget class="QTextEdit" name="textEditX">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditY">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditZ">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditRX">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditRY">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditRZ">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RZ</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupGoalPulse">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>280</y>
      <width>191</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Goal Pulse</string>
    </property>
    <widget class="QTextEdit" name="textEditR5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R2</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditR2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditR1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditR4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditR6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R1</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditR3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R5</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxVel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>560</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Velocity</string>
    </property>
    <widget class="QTextEdit" name="textEditV">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>1161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Time</string>
    </property>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send and Receive Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>40</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserTime">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1206</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
